api/src/main/java/org/apache/iceberg/transforms/Transform.java

```diff
@@ -58,6 +58,29 @@
    */
   Type getResultType(Type sourceType);
 
+  /**
+   * Whether the transform preserves the order of values (is monotonic).
+   * <p>
+   * A transform preserves order for values when for any given a and b, if a &lt; b then apply(a) &lt;= apply(b).
+   *
+   * @return true if the transform preserves the order of values
+   */
+  default boolean preservesOrder() {
+    return false;
+  }
+
+  /**
+   * Whether ordering by this transform's result satisfies the ordering of another transform's result.
+   * <p>
+   * For example, sorting by day(ts) will produce an ordering that is also by month(ts) or year(ts). However, sorting
+   * by day(ts) will not satisfy the order of hour(ts) or identity(ts).
+   *
+   * @return true if ordering by this transform is equivalent to ordering by the other transform
```

Comment:
"equivalent to" -> I thought it would be "satisfies"?

Reply:
I think "equivalent to" is more specific in this case because "satisfies" is in the method name. I'm trying to avoid using the term in the definition, and the meaning of "satisfies" is that ordering by one is equivalent to ordering by the other.

Reply:
Makes sense, it's the orderings that are equivalent.